pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -964,4 +963,128 @@ self: super: {
   # https://github.com/danidiaz/pipes-transduce/issues/2
   pipes-transduce = super.pipes-transduce.override { foldl = self.foldl_1_1_6; };
 
+  # Haste stuff:
```

Comment:
IMHO, the `configuration-common.nix` is a really poor choice for putting documentation there. How about `doc/haskell-users-guide.md` instead? Or maybe just mention that stuff in the commit message?